src/pagination.js

```diff
@@ -17,12 +17,17 @@ export default React.createClass({
     this.setState({page: nextProps.page})
   },
   getSafePage (page) {
+    if(isNaN(page)) {
```

Comment:
This fixes the warning you'd get if typing two non-numeric characters, which caused page to be NaN